int16 distanceX
int16 distanceY
int16 obstacleX
int16 obstacleY
int16 obstacleXMin
int16 obstacleXMax
int16 obstacleYMin
int16 obstacleYMax
bool startobs_DMflag
